components/profile/overview/regularity/Banner/Banner.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-nested-ternary */
+
+import clsx from 'clsx'
+
+import BasePlayerIcon from '@/icons/banner/basePlayerIcon.svg'
+import FirstElevenIcon from '@/icons/banner/firstElevenIcon.svg'
+import GoodReplacementIcon from '@/icons/banner/goodReplacementIcon.svg'
+import ReservePlayerIcon from '@/icons/banner/reservePlayerIcon.svg'
+
+import { usePlayerRole } from './hooks'
+
+interface IProps {
+  playerName: string
+  firstElevenPercent: number
+  fromBenchPercent: number
+  benchPercent: number
+}
+
+export const Banner = ({
```

Comment:
we need something more descriptive
```suggestion
export const PlayerRegularityBanner = ({
```